packages/sanity/src/core/store/user/hooks.ts

```diff
@@ -22,7 +22,22 @@ export function useUser(userId: string): LoadingTuple<User | null | undefined> {
   return useUserViaUserStore(useMemo(() => [userStore, userId], [userId, userStore]))
 }
 
-/** @internal */
+/**
+ * Retrieves the current user from the source.
```

Comment:
To third party developers, I don't think it is clear what a "source" is, and as we don't really utilize the concept for anything at the moment, I think we should steer away from using it.

```suggestion
 * Retrieves information about the currently authenticated user.
```